xcore/interface/stitcher.cpp

```diff
@@ -619,6 +617,8 @@ BowlModel::get_bowlview_vertex_model (
         PointFloat2 texture_pos1;
         if (row + 1 <= res_height) {
             texture_pos1.y = (row + 1) * step_y;
+        } else {
+            texture_pos1.y = 0.0f;
```

Comment:
if/else should be reduced in loop as possible as you can.

Why not change the for loop to 

       for (row=0; row < res_height -1;  row++)
 and remove condition check of

        if (row+1) 
in loop body.
This need a XCAM_FAIL_RETURN check to make sure res_height >=2.